src/meltano/core/plugin/singer/tap.py

```diff
@@ -185,6 +185,9 @@ class SingerTap(SingerPlugin):  # noqa: WPS214
             value_processor="nest_object",
         ),
         SettingDefinition(name="_select_filter", kind=SettingKind.ARRAY, value=[]),
+        SettingDefinition(
+            name="_no_catalog_cache", kind=SettingKind.BOOLEAN, value=False
+        ),
```

Comment:
Can you add documentation for this?

https://docs.meltano.com/concepts/plugins/#extras would be a good place.

Reply:
added